<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>isotopinator</class>
 <widget class="QWidget" name="isotopinator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout" columnstretch="1,1,1">
       <item row="1" column="0">
        <widget class="QLineEdit" name="mass_split_input">
         <property name="placeholderText">
          <string>1.003355</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="search_mass_split_btn">
         <property name="text">
          <string>Search split</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="mass_split_label">
         <property name="text">
          <string>Mass split (Da)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="margin">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="tolerance_label">
         <property name="text">
          <string>Tolerance (Da)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="margin">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="tolerance_input">
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>0.0001</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item row="1" column="0">
        <widget class="QListWidget" name="candidate_list">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="candidate_label">
         <property name="text">
          <string>Candidate split</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="margin">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="error_label">
         <property name="text">
          <string>Error (Da)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="margin">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QListWidget" name="error_list">
         <property name="font">
          <font>
           <family>Courier</family>
           <pointsize>14</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="isotopinator_tips_label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Tips 1 : Only the first 7 d√©cimals of the error are displayed&lt;/p&gt;&lt;p&gt;Tips 2 : Results are sorted from the lowest absolute error to the largest.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="margin">
        <number>10</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
